sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -1474,6 +1474,34 @@ case class Pivot(
   override protected def withNewChildInternal(newChild: LogicalPlan): Pivot = copy(child = newChild)
 }
 
+/**
+ * A constructor for creating a transpose, which will later be converted
```

Comment:
A constructor? It's a normal logical plan node.

Reply:
Adjusted. I will file a follow-up to remove other "A constructor for ..." in the file :)